onlinecasino/src/main/java/org/example/model/BlackJack.java

```diff
@@ -0,0 +1,13 @@
+package org.example.model;
+
+import java.math.BigDecimal;
+import java.util.Set;
+
+public class BlackJack extends GameNeedDealer{
+    private static final int BLACK_JACK_AMOUNT_OF_PLAYERS = 6;
+    private static final BigDecimal BLACKJACK_DAILY_REVENUE = BigDecimal.valueOf(1000000);
+    private static final Set<Experience> NEEDED_EXPERIENCE_FOR_BLACKJACK = Set.of(Experience.NEWBIE, Experience.MID_LEVEL, Experience.PRO);
```

Comment:
I suggest making the dealer class a parent with a child class for the pro dealer so you can pass these with the typesafety ensured